compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaTupleMemberSymbol.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.ballerina.compiler.api.impl.symbols;
+
+import io.ballerina.compiler.api.impl.SymbolFactory;
+import io.ballerina.compiler.api.symbols.AnnotationAttachmentSymbol;
+import io.ballerina.compiler.api.symbols.AnnotationSymbol;
+import io.ballerina.compiler.api.symbols.SymbolKind;
+import io.ballerina.compiler.api.symbols.TupleMemberTypeSymbol;
+import io.ballerina.compiler.api.symbols.TypeSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationAttachmentSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BVarSymbol;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Represents a tuple member type symbol.
```

Comment:
```suggestion
 * Represents a tuple member symbol.
```

Reply:
+1

Reply:
```ballerina
[string, int, boolean] values = ["AA", 10, true];
```

Here, what we call `string` in `[string, int, boolean]` and `"AA"` in `["AA", 10, true]`?

Reply:
Consider `[@annot string, int]`.

If we consider the first member
- tuple member - `@annot string`
- tuple member type - `string`

If there are no annotations, they both look the same. But the member is annotations + type whereas the type is just the type.

Reply:
The RHS in your example is just a list constructor, right? So maybe list member ("AA" specifically single list member). https://ballerina.io/spec/lang/master/#list-constructor-expr

Reply:
Ah got it. Thanks for the explanation.